python/hail/matrixtable.py

```diff
@@ -2307,42 +2307,39 @@ def format_type(typ):
                                                               e=entry_fields)
         print(s)
 
-    @typecheck_method(order=listof(str))
-    def reorder_columns(self, order: List[str]) -> 'MatrixTable':
-        """Reorder columns.
-
-        .. include:: _templates/req_tstring.rst
+    @typecheck_method(indices=listof(int))
+    def choose_cols(self, indices: List[int]) -> 'MatrixTable':
+        """Choose a new set of columns from a list of old column indices.
```

Comment:
What happens if indices are repeated? I would expect to end up with columns which are independent copies (i.e. deep copies) of the same source column. If that works, should probably add a test to cover that case. If not, we should document the precondition and add error reporting.

Reply:
totally OK! I'll add a test.